frontend/components/forms/FormField/_styles.scss

```diff
@@ -2,11 +2,12 @@
   margin-bottom: $pad-large;
 
   &__label {
-    font-size: $x-small;
+    font-size: $small;
     font-weight: $bold;
     color: $core-fleet-black;
     display: block;
-    margin-bottom: $pad-xsmall;
+    margin-bottom: $pad-small;
+    height: $form-field-label-height;
```

Comment:
Is there a way we can make all the paddings match between the text and the dashed underline?
<img width="623" alt="Screenshot 2022-11-22 at 2 30 33 PM" src="https://user-images.githubusercontent.com/71795832/203405119-8a9e206a-bceb-46cf-81fa-5ad70cbc627e.png">